test/maki.test.js

```diff
@@ -142,3 +146,23 @@ test('valid svgs ', function(t) {
     );
   }
 });
+
+test('svg are compatible with style components', (t) => {
```

Comment:
I'm concerned that this test will be opaque in intention to folks outside of the organization. Can we give more detail to the requirements here, so external contributors are able to troubleshoot?